hw3/01-characters.js

```diff
@@ -0,0 +1,33 @@
+const character = document.getElementsByClassName("character");
+
+async function getCharacters() {
+  const url = "https://thronesapi.com/api/v2/Characters";
+  try {
+    let response = await fetch(url);
+    return await response.json();
+  } catch (erorr) {
+    console.log(error);
+  }
+}
+
+async function renderCharacters() {
+  let characters = await getCharacters();
+  let html = "";
+
+  characters.forEach((character) => {
+    let htmlSegment = `<div class="character" onmouseover="this.style.background='var(--blue)';" onmouseout="this.style.background='none';">
+                        <div style="margin:1em;">
+                          <img src="${character.imageUrl}" >
+                          <h3>${character.firstName} ${character.lastName}</h3>
+                          <h4>${character.title}</h4>
+                        </div>
+                      </div>`;
+
+    html += htmlSegment;
+  });
+
+  let section = document.querySelector("section");
```

Comment:
Consider using a class/id for the query selector, and not just selecting the first section tag. This would avoid any ambiguity if for example another "section" element was added to the page.